src/redux/actions/userActions.js

```diff
@@ -113,9 +112,7 @@ export const loginUserAsync = (
 
     try {
       // check if email exists in db
-      user = await axios.get(
-        `http://localhost:3001/api/users/findUser/${email}`
-      );
+      user = await axios.get(`${BASE_URL}/findUser/${email}`);
```

Comment:
im assuming the base url is like http://localhost:3000/api so here you're missing the /users part of the path